src/user-meta/user-interface/custom-meta-integration.php

```diff
@@ -78,4 +85,35 @@ public function process_user_option_update( $user_id ) {
 			\delete_user_meta( $user_id, $meta->get_key() );
 		}
 	}
+
+	/**
+	 * Adds the inputs needed for SEO values to the User Profile page.
+	 *
+	 * @param WP_User $user User instance to output for.
+	 *
+	 * @return void
+	 */
+	public function user_profile( $user ) {
+		\wp_nonce_field( 'wpseo_user_profile_update', 'wpseo_nonce' );
+
+		/* translators: %1$s expands to Yoast SEO */
+		$wpseo_up_settings_header = \sprintf( \__( '%1$s settings', 'wordpress-seo' ), 'Yoast SEO' );
```

Comment:
The `wpseo_up_settings_header` name is a bit confusing to me. Specifically, the `up` part. What am I missing?
Ohh, `User Profile`?

Reply:
This is a residual from how it was in the deprecated file where I moved all this from. But you're right, I'll rename it into something more coherent.

Reply:
[Done](https://github.com/Yoast/wordpress-seo/pull/21314/commits/83f4be55fb1d42260257d91eba97bd223c1f43d0)